python/cudf/cudf/_lib/cpp/io/orc.pxd

```diff
@@ -37,7 +37,6 @@ cdef extern from "cudf/io/orc.hpp" \
         void enable_use_np_dtypes(bool val) except+
         void set_timestamp_type(data_type type) except+
         void set_decimal_cols_as_float(vector[string] val) except+
-        void enable_decimal128(bool val) except+
```

Comment:
Was this one missed? https://github.com/rapidsai/cudf/blob/385984a67991ff481905950abfc5ff0fb33c6011/python/cudf/cudf/_lib/cpp/io/orc.pxd#L61